pygmt/src/ternary.py

```diff
@@ -52,18 +52,18 @@ def ternary(self, data, **kwargs):
         [*amin*, *amax*, *bmin*, *bmax*, *cmin*, *cmax*].
         Give the min and max limits for each of the three axes **a**, **b**,
         and **c**.
-    {CPT}
-    {G}
+    {cmap}
+    {fill}
```

Comment:
```suggestion
    {color}
```
Fixes https://github.com/GenericMappingTools/pygmt/actions/runs/3067316855/jobs/4953471467#step:11:28
```python-traceback
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/runner/work/pygmt/pygmt/pygmt/__init__.py", line 25, in <module>
    from pygmt import datasets
  File "/home/runner/work/pygmt/pygmt/pygmt/datasets/__init__.py", line 5, in <module>
    from pygmt.datasets.earth_age import load_earth_age
  File "/home/runner/work/pygmt/pygmt/pygmt/datasets/earth_age.py", line 10, in <module>
    from pygmt.src import grdcut, which
  File "/home/runner/work/pygmt/pygmt/pygmt/src/__init__.py", line 52, in <module>
    from pygmt.src.ternary import ternary
  File "/home/runner/work/pygmt/pygmt/pygmt/src/ternary.py", line 26, in <module>
    def ternary(self, data, **kwargs):
  File "/home/runner/work/pygmt/pygmt/pygmt/helpers/decorators.py", line 481, in fmt_docstring
    module_func.__doc__ = docstring.format(**filler_text)
KeyError: 'fill'
```

Reply:
Oops nevermind, you beat me to it :laughing: